etl/pad/raw/skills/emoji_en/enemy_skill_text.py

```diff
@@ -0,0 +1,599 @@
+from pad.raw.skills.emoji_en.skill_common import *
+
+import logging
+
+human_fix_logger = logging.getLogger('human_fix')
```

Comment:
Can the emoji additions happen in pad-cogs? either no emoji addition here at all or some kind of text replacement that we can replace in pad-cogs

Reply:
I can just use discord aliases like ":thumbsup:" or {thumbsup} depending on whichever one is safer

Reply:
I left more comments in discord, but I don't think emojis period are appropriate here